### PR TITLE
Add VPC support, disable master auto-provisioning, etc.

### PR DESCRIPTION
**Pull request by legal90**
- Disable auto-provisioning of the master with a test job.
- Add VPC support. Instances will be created in the VPC specified by a variable.
- Fix provisioning of the master node. JRE 8 is required.
- Update defaults for variables, fix alignment.

### CODE REVIEW

File: security_group.tf [hunk 1]
```diff
@@ -1,43 +1,44 @@
...
 resource "aws_security_group_rule" "ingress-public-sgr" {
-  type = "ingress"
-  from_port = 22
-  to_port = 22
-  protocol = "tcp"
+  type        = "ingress"
+  from_port   = 22
+  to_port     = 22
+  protocol    = "tcp"
   cidr_blocks = ["0.0.0.0/0"]
```
Comment (pydavid) [inline security_group.tf]: 0.0.0.0/0 is too open. 
Can you put the public ip  ? `curl ifconfig.co`

File: variables.tf [hunk 2]
```diff
@@ -1,50 +1,48 @@
...
+variable "subnet_ids" {
+  description = "The list of subnet IDs where the cluster will be created. Master node will be created in teh first subnet mentioned in this list"
+  type        = "list"
+}
+
 variable "aws_amis" {
   default = {
-    "us-east-1" = "ami-6869aa05"
-    "us-west-2" = "ami-7172b611"
+    "us-west-2" = "ami-39d39d41"
```
Comment (pydavid) [inline variables.tf]: can you add a comment to say that it's the 2018 AWS official AMI ?

**Comment by pydavid** [comment]
- Can you update the README.md 
-  Create a `main.tf`

**Comment by legal90** [comment]
@pydavid Thanks! I've done some refactoring, moved security groups out of this module's scope and added `main.tf` as a placeholder with a text reference to `master.tf` and `slave.tf` (there are no common resources for now).

Please, review it again

**Review by pydavid — APPROVED** [review]

**Pull request merged**